README.md

```diff
@@ -10,76 +10,34 @@ Active Admin is a Ruby on Rails framework for creating elegant backends for webs
 
 ## State of the project
 
-### 1.0.0
+1.0.0.pre5 is the last release to support Rails 3.2 and Ruby 1.9.3.
+It is provided to help with upgrades to Rails 4 only.
 
-We're [currently working on 1.0.0](https://github.com/activeadmin/activeadmin/issues?milestone=18),
-which as far as dependencies, moves us from meta_search to Ransack and adds Rails 4 & 5 support.
-
-You can get it by tracking master:
-```ruby
-gem 'activeadmin', git: 'https://github.com/activeadmin/activeadmin'
-```
-
-Or you can using rubygems:
-```ruby
-gem 'activeadmin', '~> 1.0.0.pre4'
-```
-
-*Keep in mind that during the time where we use `pre`-release label, things can break in each release!*
-
-### Rails 5
-
-Active Admin master has preliminary support for Rails 5. To give it a try, these Gemfile changes may be needed:
-
-```ruby
-gem 'inherited_resources', git: 'https://github.com/activeadmin/inherited_resources'
-```
-
-### 0.6.x
-
-The plan is to follow [semantic versioning](http://semver.org/) as of 1.0.0. The 0.6.x line will
-still be maintained, and we will backport bug fixes into future 0.6.x releases. If you don't want
-to have to wait for a release, you can track the branch instead:
-
-```ruby
-gem 'activeadmin', git: 'https://github.com/activeadmin/activeadmin', branch: '0-6-stable'
-```
-
-## Documentation
-
-Please note that <http://activeadmin.info> is out of date. For the latest docs, check out the
-Github [docs](https://github.com/activeadmin/activeadmin/tree/master/docs#activeadmin-documentation) and the [wiki](https://github.com/activeadmin/activeadmin/wiki).
-
-## Links
-
-* Website: <http://www.activeadmin.info> (out of date)
-* Live demo: <http://demo.activeadmin.info/admin>
-* Documentation
-  * Guides: <https://github.com/activeadmin/activeadmin/tree/master/docs>
-  * YARD: <http://rubydoc.info/gems/activeadmin>
-  * Wiki: <https://github.com/activeadmin/activeadmin/wiki>
+1.0.0 drops support for Rails 3.2 and Ruby 1.9.3.
```

Comment:
Well, for 1.0.0 we still need, at least, to actually drop support. Maybe also releasing an rc version and fixing last minute bugs. I'd say let's do one thing at a time ;)